runtime/build.gradle.kts

```diff
@@ -18,45 +16,80 @@ kotlin {
         nodejs {}
     }
 
+    val onPhone = System.getenv("SDK_NAME")?.startsWith("iphoneos")?:false
+    if(onPhone){
+        iosArm64("ios")
+    }else{
+        iosX64("ios")
+    }
+
     // For ARM, should be changed to iosArm32 or iosArm64
     // For Linux, should be changed to e.g. linuxX64
     // For MacOS, should be changed to e.g. macosX64
     // For Windows, should be changed to e.g. mingwX64
-    // macosX64("macos")
+    macosX64()
 
-    sourceSets["commonMain"].dependencies {
-        implementation(kotlin("stdlib-common"))
-        implementation("org.jetbrains.kotlinx:kotlinx-serialization-runtime-common:${Versions.kotlinSerialization}")
-    }
-    sourceSets["commonTest"].dependencies {
-        implementation(kotlin("test-common"))
-        implementation(kotlin("test-annotations-common"))
-    }
+    sourceSets.create("nativeMain")
```

Comment:
I think the typical way to write this is:

```kt
sourceSets {
    val nativeMain by creating {
        ...
    }
}
```

Reply:
Indeed, thanks!